components/sidebar.jsx

```diff
@@ -0,0 +1,1077 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This whole file was removed and replaced by @jespino with the ones at `components/sidebar/` when he split it up into more components, so we can probably get rid of this one